src/lib/wind_estimator/WindEstimator.cpp

```diff
@@ -39,50 +39,52 @@
 #include "WindEstimator.hpp"
 
 bool
-WindEstimator::initialise(const matrix::Vector3f &velI, const matrix::Vector2f &velIvar, const float tas_meas,
-			  const matrix::Quatf &q_att)
+WindEstimator::initialise(const matrix::Vector3f &velI, const float hor_vel_variance, const float heading_rad,
+			  const float tas_meas, const float tas_variance)
 {
 	// do no initialise if ground velocity is low
 	// this should prevent the filter from initialising on the ground
 	if (sqrtf(velI(0) * velI(0) + velI(1) * velI(1)) < 3.0f) {
```

Comment:
pls constexpr this `3.0f`.

What was the process in choosing this value? Would be awesome (not for this PR) if we could show some data on threshold selection. -- just thinking of the case where a fixed-wing has just taken off the ground measuring 10m/s airspeed but traveling relative to the ground at less than 3m/s -- what happens? I've had this case many times tossing easy gliders into the air with high wind, they go up like helicopters :)

Reply:
Remove this check from wind estimator, check with unit tests what happens when ground speed is zero

Reply:
Did you check what the effect is when ground speed is 0?

Reply:
@sfuhrer Yes, there is no singularity that needs to be handled when ground speed is zero. In that case wind will be equal to airspeed, which is what you would expect.